webpack.config.cjs

```diff
@@ -230,6 +230,87 @@ const storyEmbedBlock = {
   },
 };
 
+const webStoriesBlock = {
+  ...sharedConfig,
+  entry: {
+    'web-stories-block': './assets/src/web-stories-block/index.js',
+  },
+  module: {
+    rules: [
+      !isProduction && {
+        test: /\.js$/,
+        use: ['source-map-loader'],
+        enforce: 'pre',
+      },
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
+        use: [
+          require.resolve('thread-loader'),
+          {
+            loader: require.resolve('babel-loader'),
+            options: {
+              // Babel uses a directory within local node_modules
+              // by default. Use the environment variable option
+              // to enable more persistent caching.
+              cacheDirectory: process.env.BABEL_CACHE_DIRECTORY || true,
+            },
+          },
+        ],
+      },
+      {
+        test: /\.svg$/,
+        use: [
+          {
+            loader: '@svgr/webpack',
+            options: {
+              titleProp: true,
+            },
+          },
+          'url-loader',
+        ],
+      },
```

Comment:
Is this already part of the shared config?
https://github.com/rtCamp/web-stories-wp/blob/e1138ce9d0eed8f0d41230cf7232a58943e57833/webpack.config.cjs#L91-L102

Reply:
Yes, since we are using the `postcss-loader` on `.css` files if I use the `sharedConfig.module.rules` it throws error in build as there are conflicts in the defined loaders for the `/.css$/`. 

So I had just picked the rules part from the shared config.